O/OpenSourceRoutingMachine/build_tarballs.jl

```diff
@@ -0,0 +1,157 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OpenSourceRoutingMachine"
+version = v"6.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Project-OSRM/osrm-backend.git", "054b0a6395f689a47a908b26b12e32ed7704c533"),
+    DirectorySource("./bundled"),
+    # OSRM requires C++20, which needs a newer SDK
+    ArchiveSource("https://github.com/realjf/MacOSX-SDKs/releases/download/v0.0.1/MacOSX12.3.sdk.tar.xz",
+                  "a511c1cf1ebfe6fe3b8ec005374b9c05e89ac28b3d4eb468873f59800c02b030"),    
```

Comment:
Should be switched to the `require_macos_sdk` helper, see e.g. fddd43fd1542cb61c57d16dc12305284aa344ce4 for examples how to do that.